tasks/brew.yml

```diff
@@ -1,8 +1,7 @@
 ---
-- name: install casks
+- name: install brew package
   become: '{{ kubernetes_helm_privilege_escalate | default(False) }}'
   become_user: root
-  with_items: '{{ kubernetes_helm_casks | default(["helm"]) }}'
-  homebrew_cask:
-    name: '{{ item }}'
+  homebrew:
+    name: '{{ kubernetes_helm_brew_pakage }}'
```

Comment:
c's are free. u only have to buy vowels.

Reply:
typo fixed